lib/utils/getFreePort.ts

```diff
@@ -1,20 +1,27 @@
+import * as net from 'net';
+
 /**
  * @method getFreePort
  * @param host
  * @returns {Promise<number>}
  */
-module.exports = function(host = 'localhost') {
-  const net = require('net');
-
+export = function(host = 'localhost'): Promise<number> {
```

Comment:
Since in `lib/transport/selenium-webdriver/service-builders/appium.js`, we are destructing `getFreePort`, so it make sense to export `getFreePort` function instead?
Something like this:
```js
export function getFreePort(host = 'localhost'): Promise<number>
```

Reply:
`appium.js` is getting the `getFreePort` from `utils/index.js` in the latter file we've used `const getFreePort = require('./getFreePort')`, so the export statement won't work here, not until we migrate `utils/index.js` to TS.